LoopKitUI/Views/Settings Editors/TherapySettingsViewModel.swift

```diff
@@ -111,3 +116,58 @@ public class TherapySettingsViewModel: ObservableObject {
         didSave?(TherapySetting.insulinSensitivity, therapySettings)
     }
 }
+
+// MARK: Navigation
```

Comment:
👍 for moving this to the view model!  Now that it's here, would be nice to see unit tests for it... :)